docs/user-guide.md

```diff
@@ -137,7 +137,8 @@ strategies:
 Descheduler's `RemovePodsViolatingNodeTaints` strategy can be combined with
 [Node Problem Detector](https://github.com/kubernetes/node-problem-detector/) and
 [Cluster Autoscaler](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler) to automatically remove
-Nodes which have problems. Node Problem Detector can detect specific Node problems and taint any Nodes which have those
+Nodes which have problems. Node Problem Detector can detect specific Node problems and reports them to apiserver.
```

Comment:
nit: `and report them to the API server`

Reply:
fix done~ please check :)